docs/topics/jobs.md

```diff
@@ -0,0 +1,45 @@
+# Jobs
+
+Looking for a Django REST Framework related job? We highlight job vacancies by our awesome sponsors here and also provide a list of other job resources that may be helpful.
+
+
+## Sponsors Who Are Hiring
```

Comment:
I reckon the easy path for us to take here would be *not* to highlight sponsors who're known to be hiring, but rather to simply note "if you're looking for a new role, it's worth checking out if any of our sponsors are hiring" (and then either include the sponsor pics, or just link to the funding site)

We *could* later ask our sponsors if they'd prefer a more *active* approach on this page, but the simple tack is probably a good starting point.